shared/mainnet/config.yaml

```diff
@@ -39,8 +39,11 @@ BELLATRIX_FORK_EPOCH: 144896 # Sept 6, 2022, 11:34:47am UTC
 # Capella
 CAPELLA_FORK_VERSION: 0x03000000
 CAPELLA_FORK_EPOCH: 18446744073709551615
+# EIP4844
+EIP4844_FORK_VERSION: 0x04000000
```

Comment:
how about we use `0x48444844` for version until it has settled?

Reply:
I think that's reasonable, to avoid conflicts if nothing else

Reply:
ie as a general principle, don't assign a version number until a feature has been scheduled